src/transformers/pipelines/visual_question_answering.py

```diff
@@ -10,7 +10,10 @@
     from ..image_utils import load_image
 
 if is_torch_available():
-    from ..models.auto.modeling_auto import MODEL_FOR_VISUAL_QUESTION_ANSWERING_MAPPING_NAMES
+    from ..models.auto.modeling_auto import (
+        MODEL_FOR_VISUAL_QUESTION_ANSWERING_MAPPING_NAMES,
+    )
```

Comment:
nit - can stay on one line 

```suggestion
    from ..models.auto.modeling_auto import MODEL_FOR_VISUAL_QUESTION_ANSWERING_MAPPING_NAMES
```

Reply:
Fixed! Ruff autoformatting loves breaking things into multiple lines :)